examples/models/lightgbm_custom_server/artifacts/model-settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "irislgbm",
+    "implementation": "mlserver_lightgbm.LightGBMModel",
```

Comment:
@cliveseldon  Is Seldon going to support ligthtgbm Pre-Packaged Inference Servers in the next release?

Reply:
we are looking to make it official but for now this example shows how to do it yourself.

Reply:
I see that LIGHTGBM_SERVER is already being used here[1], it's already supported in the dev version and doesn't need to be customized anymore, right?

[1] https://github.com/SeldonIO/seldon-core/blob/master/examples/models/lightgbm_custom_server/model_seldon_v2.yaml#L9

Reply:
No this is part of an example of how to create custom prepackaged servers: https://docs.seldon.io/projects/seldon-core/en/latest/examples/custom_server.html

Reply:
I see, you still need to register lightgbm server, but I see that the example seldon has already packaged its own image of lightgbm server, so this is currently available, right?
https://hub.docker.com/r/seldonio/lighgbmserver

Reply:
Does this mirror use mlserver[1] under the hood?

[1] https://github.com/SeldonIO/MLServer

Reply:
for the v2 protocol yes